src/Common.OData.ApiExplorer/AspNet.OData/Builder/DefaultODataQueryOptionDescriptionProvider.cs

```diff
@@ -242,7 +242,8 @@ static void AppendAllowedOptions( StringBuilder description, ODataQueryOptionDes
             }
 
             if ( context.AllowedFunctions != AllowedFunctions.None &&
-                 context.AllowedFunctions != AllowedFunctions.All )
+                 context.AllowedFunctions != AllowedFunctions.All &&
```

Comment:
I don't _think_ this line is required at all anymore. I didn't even realize that `all` is a function. I'm pretty sure I mistook `All` for `AllFunctions`. If things still work with only `AllFunctions`, then we should be good.

Reply:
I removed the 'All' check. See the changed test on how it affects the texts.

As it's possible for the user to say `[EnableQuery(AllowedFunctions = AllowedFunctions.All)]`, maybe it would be better to swap to the `HasFlag(AllowedFunctions.All)` approach?